Objects/Converters/ConverterNavisworks/ConverterNavisworks/ConverterNavisworks.Geometry.cs

```diff
@@ -485,3 +485,13 @@ private static IEnumerable<double> MoveAndScaleVertices(Vector3D vertex1, Vector
     }
   }
 }
+
+namespace Objects.Organization
+{
+  public class Geometry : Base
```

Comment:
What is this class for? If it's empty why not use a `Base` instead?

Since this is only available in Navis, it will become a `Base` when received, since the Kit doesn't know about this type, and the deserialisation picks from the Kit classes only.

Reply:
Placeholder! Will be moved to Objects when it's ready.

Since Navis doesn't receive, deserialisation to Base is a non-issue now.